src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -29,6 +29,9 @@
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
     public static final String MESSAGE_INSUFFICIENT_PARTS = "Number of parts must be more than 1.";
+    private static final Phone unspecifiedPhone = new Phone();
```

Comment:
These should be in their respective field classes (`Phone.UNSPECIFIED`, `Address.UNSPECIFIED`, etc.)

Reply:
not sure how to implement it the way u suggest, ive already import Phone but having trouble with some syntax implementations